client/src/App.js

```diff
@@ -1,27 +1,65 @@
-import logo from "./logo.svg";
+import React from "react";
 import "./App.css";
 import Button from "./components/Button";
+import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";
 
 function App() {
+  //basic examples to test the Routing
+
+  function Survey() {
+    return <h2>Survey</h2>;
+  }
+
+  function Whiteboard() {
+    return <h2>Whiteboard</h2>;
+  }
+
+  function Dashboard() {
+    return <h2>Dashboard</h2>;
+  }
+
+  function Login() {
+    return <h2>Login</h2>;
+  }
```

Comment:
I know that this is only temporary, but try not to create components inside other components.
Thus, move these components outside of App (or in a different file -> You already created these files).

Otherwise, you will create a new component on every render which will lead to performance issues.

Reply:
Will do, thanks for the review! :)